scripts/troubleshoot/LogCollection/README.md

```diff
@@ -0,0 +1,41 @@
+# Container Insights Log collector
+
+This tool will collect:
+* agent logs from linux ds and rs pods;
+* agent logs from windows pod if enabled;
+* cluster/node info, pod deployment, configMap, process logs etc..
+
+## Prerequisites
+* kubectl: az aks install-cli
+* tar (installed by default)
+* all nodes should be running on AKS
+* AKS Insights are enabled: https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-onboard
+
+Otherwise, script will report error message and exit.
+
+## How to run
+```
+wget https://raw.githubusercontent.com/MSNina123456/ContainerInsightsLog/main/AKSInsightsLogCollection.sh && bash ./AKSInsightsLogCollection.sh
```

Comment:
update this link to ci_dev branch otherwise once the PR merged, this will be broken link.

Reply:
updated